tests/testthat/test-f7VirtualList.R

```diff
@@ -0,0 +1,20 @@
+library(shinytest2)
+test_that("virtualList works as expected", {
+  # Don't run these tests on the CRAN build servers
+  skip_on_cran()
+  shiny_app_path <-
+    system.file("examples/virtualList/app.R", package = "shinyMobile")
+  app <- AppDriver$new(
+    shiny_app_path,
+    name = "virtualList-app",
+    variant = platform_variant()
+  )
+  # Animation/transition takes a bit of time
+  app$wait_for_idle(1000)
+  app$expect_values(input = "vlist")
+  # Set search bar value
+  app$wait_for_js("var searchbarEl = document.getElementById('search1'); app.searchbar.search(searchbarEl, 'Title 1000')")
```

Comment:
@DivadNojnarg this is a bit tricky: snapshot png shows the difference, but the json does not since the input doesn't change whenever the results get filtered according to a search bar. Not sure if we need to worry about it now- but for this test the png does add value over the json.

Reply:
Related: since I'm running 4.2 and you 4.3 we get double snaps folders, which I find slightly annoying. We can set `r_version = FALSE` in `platform_variant`, or we have to live with it 🤔